tiledb/ml/models/pytorch.py

```diff
@@ -157,5 +156,10 @@ def _write_array(self, serialized_model_info: dict, meta: Optional[dict]):
 
             # Add extra metadata given by the user to array's metadata
             if meta:
-                for key, value in meta.items():
-                    tf_model_tiledb.meta[key] = json.dumps(value).encode("utf8")
+                self.update_model_metadata(array=tf_model_tiledb, meta=meta)
+
+            # In case we are on TileDB-Cloud we have to save model array's file properties also in metadata
+            if self.namespace:
+                self.update_model_metadata(
+                    array=tf_model_tiledb, meta=self.file_properties
+                )
```

Comment:
I'd extract in a single base `TileDBModel._update_metadata` method all the common metadata/file_properties logic currently spread in 2x3 different methods:
- `_create_array`:  `update_file_properties(self.uri, self.file_properties)`
- `_write_array`: `self.update_model_metadata(tf_model_tiledb, meta)`
- `_write_array`: `self.update_model_metadata(tf_model_tiledb, self.file_properties)`